front/next.config.js

```diff
@@ -2,7 +2,8 @@
 const nextConfig = {
     experimental: {
         typedRoutes: true,
-        serverComponentsExternalPackages: ['@prisma/client','bcrypt']
+        // serverComponentsExternalPackages lists packages that should not be called by the client (see https://youtu.be/2kgqPvs0j_I?t=1285)
```

Comment:
Is it intentional to have left this comment here?

Reply:
yes, it's not explicit in the docs that adding packages to that field ensures that those packages are only used server side